tests/benchmark/benchmark_between_releases.sh

```diff
@@ -17,19 +17,49 @@ rm Dockerfile
 echo "Successfully built images"
 
 # Create folders to hold the files
-mkdir release1
-mkdir release2
+mkdir $TAG1
```

Comment:
For consistency, can you put double quotes around all uses of $TAG1 and $TAG2 - e.g. "$TAG1".